webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultCsvValidationService.java

```diff
@@ -28,20 +29,20 @@
 @Service
 public class DefaultCsvValidationService implements CsvValidationService {
 
-    public final static String HeaderSchool = "school_natural_id";
-    public final static String HeaderSubject = "subject_code";
     public final static String HeaderGroup = "group_name";
-    public final static String HeaderUser = "group_user_login";
+    public final static String HeaderSchool = "school_natural_id";
     public final static String HeaderYear = "school_year";
+    public final static String HeaderSubject = "subject_code";
     public final static String HeaderSSID = "student_ssid";
+    public final static String HeaderUser = "group_user_login";
```

Comment:
do we plan on aliasing these so that they make more sense to a user? or do we like that they map directly to db columns for troubleshooting or something?

Reply:
They map directly to DB columns based upon index.  I suppose we could name them whatever we like, but why increase complexity for CSVs that shouldn't be hand-jammed in "real life" ?

Reply:
Oh ok. Based on the rule page i wrote i was under the impression that people would be hand jamming these to some degree but i guess we have some level of automation setup for them?

Reply:
ideally they'll be exporting them from a school system of some kind.  At least that's my understanding.

Reply:
yeah, we really do not know. we know districts export similar data to CALPADS so I would assume whoever does that could write an export to produce the CSV we have defined.